sys/dev/ixgbe/ixgbe_phy.c

```diff
@@ -1423,6 +1430,20 @@ s32 ixgbe_identify_sfp_module_generic(struct ixgbe_hw *hw)
 				else
 					hw->phy.sfp_type =
 						ixgbe_sfp_type_1g_lx_core1;
+			/*
+			 * Check for compatibility with 1000BASE-BX, which
+			 * operates at a nominal signaling rate of 1.25 Gbaud.
+			 * This rate is rounded up to 13 * 100 MBaud.
+			 */
+			} else if ((comp_codes_1g &
+			            IXGBE_SFF_BASEBX10_CAPABLE) &&
+				     (bitrate_nominal == 13)) {
```

Comment:
Are you ok dropping all this machinery around the bitrate check?  If you have the right comp_code for 1g I think it is safe to proceed and SFPs tend too often to be miscoded.  We can always add it back if there is unforeseen issue.